var/build-config.json

```diff
@@ -31,6 +31,11 @@
       "type": "gtfs",
       "feedId": "mbta-ma-us",
       "source": "${MBTA_GTFS_URL}"
+    },
+    {
+      "type": "gtfs",
+      "feedId": "mbta-ma-us-initial",
+      "source": "../disruption-free-for-dotcom.zip"
```

Comment:
Just confirming we don't have an alternate location for this yet right?

Reply:
@lemald The discussion is on the dotcom side - Walton said you all could put the disruption-free GTFS file somewhere on S3 - we just haven't figured out where yet.

Reply:
Sorry for the confusion, let's just leave this as-is for now then!